### PR TITLE
docs: add questions to FAQ

### PR DESCRIPTION
**Pull request by gatheluck**
## Issue URL

NA

## Change overview

Add followings to FAQ:
- Use Ascender without Docker
- Compatibility issue between PyTorch and Poetry
- Change the Python version to run CI jobs

## How to test

Please read the updated README.md.

## Note for reviewers

NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +146,26 @@ $ sudo dokcer compose stop
...
+We recommend using Ascender with Docker as described above. However, you might not be able to install Docker in your development environment due to permission issues or etc.
+
+In such cases, Ascender can be used without Docker. To do that, please install Poetry in your computer, and follow the steps describing in "Start development" section with ignoring the steps related to Docker.
+
+```bash
+# Install Poetry
+$ pip3 install poetry
+
+# Clone repo
+$ git clone git@github.com:cvpaperchallenge/<YOUR_REPO_NAME>.git
```
Comment (narusemotoki) [inline README.md:160]:
```suggestion
$ git clone git@github.com:<YOUR_USER_NAME>/<YOUR_REPO_NAME>.git
```
IIUC, Ascender is used as template project. If so actual project is in under user repositories.

File: README.md [hunk 2]
```diff
@@ -144,6 +146,26 @@ $ sudo dokcer compose stop
 
 ## FAQ
 
+### Use Ascender without Docker
+
+We recommend using Ascender with Docker as described above. However, you might not be able to install Docker in your development environment due to permission issues or etc.
```
Comment (narusemotoki) [inline README.md:151]: Does `described above` mean `- **Container**: Use of [Docker](https://www.docker.com/) reduces development environment dependencies and improves code portability.`?

File: README.md [hunk 3]
```diff
@@ -162,4 +184,22 @@ $ id -u $USER  # check UID
 $ id -g $USER  # check GID
 ```
 
-In Ascender, default value of both is `1000`. If UID or GID of your local PC is not `1000`, you need to modify the value of `UID` or `GID` inside of `docker-compose.yaml` to align your local PC (please edit their values from `1000`).
\ No newline at end of file
+In Ascender, default value of both is `1000`. If UID or GID of your local PC is not `1000`, you need to modify the value of `UID` or `GID` inside of `docker-compose.yaml` to align your local PC (please edit their values from `1000`).
```
Comment (narusemotoki) [inline README.md:187]:
```suggestion
In Ascender, default value of both are `1000`. If UID or GID of your local PC is not `1000`, you need to modify the value of `UID` or `GID` inside of `docker-compose.yaml` to align your local PC (please edit their values from `1000`).
```

I'm not sure.

File: README.md [hunk 4]
```diff
@@ -162,4 +184,22 @@ $ id -u $USER  # check UID
...
+
+### Compatibility issue between PyTorch and Poetry
```
Comment (narusemotoki) [inline README.md:189]: This is about English. I didn't know this usage of `Compatibility`. I only knew `Compatibility` kind of `Replacable`. Thank you.

File: README.md [hunk 5]
```diff
@@ -162,4 +184,22 @@ $ id -u $USER  # check UID
...
+
+Currently, there is a compatibility issue between PyTorch and Poetry. This issue is being worked on by the Poetry community and is expected to be resolved in 1.2.0. (You can check pre-release of 1.2.0 from [here](https://github.com/python-poetry/poetry/releases/tag/1.2.0b3).) 
+
+We plan to incorporate Poetry 1.2.0 into Ascender immediately after its release. In the meantime, please consider using the workaround described in [this issue](https://github.com/python-poetry/poetry/issues/4231).
```
Comment (narusemotoki) [inline README.md:193]:
```suggestion
We plan to incorporate Poetry 1.2.0 into Ascender immediately after its released. In the meantime, please consider using the workaround described in [this issue](https://github.com/python-poetry/poetry/issues/4231).
```
I'm not sure. I know I'm bad English speaker.

**Review by narusemotoki — APPROVED** [review]
LGTM
Please ignore my comment for English.

File: README.md (hunk 3 above)
Comment (gatheluck) [reply README.md:187]: Thank you! I think in this case subject is "value" (single) so maybe "is" will be correct. (Both Google translate and DeepL recommend "is" in my env. So maybe correct.)

File: README.md (hunk 2 above)
Comment (gatheluck) [reply README.md:151]: Yes, it is correct. And it is also explained in [the slide format resources about Ascender (Japanese)](https://cvpaperchallenge.github.io/Britannica/ascender/ja).

File: README.md (hunk 5 above)
Comment (gatheluck) [reply README.md:193]: Here "its release" is used as noun. So release will be appropriate.

**Pull request merged**